src/azure-cli/azure/cli/command_modules/acr/import.py

```diff
@@ -86,7 +93,39 @@ def acr_import(cmd,
                                               target_tags=target_tags,
                                               untagged_target_repositories=repository,
                                               mode=ImportMode.force.value if force else ImportMode.no_force.value)
-    return client.import_image(
+    result_poller = client.import_image(
         resource_group_name=resource_group_name,
         registry_name=registry_name,
         parameters=import_parameters)
+
+    return _handle_result(cmd, result_poller, source_registry, source_image, registry)
+
+
+def _handle_result(cmd, result_poller, source_registry, source_image, registry):
+    try:
+        result = LongRunningOperation(cmd.cli_ctx, 'Importing image...')(result_poller)
+    except CLIError as e:
+        try:
+            # if command fails, it might be because user specified registry twice in --source and --registry
+            if source_registry:
+
+                if not hasattr(registry, 'login_server'):
+                    if is_valid_resource_id(source_registry):
+                        registry, _ = get_registry_by_name(cmd.cli_ctx, parse_resource_id(source_registry)["name"])
+                    else:
+                        registry = get_registry_from_name_or_login_server(cmd.cli_ctx, source_registry, source_registry)
+
+                if registry.login_server.lower() in source_image.lower():
+                    logger.warning("\nImport failed. Please check that fully qualified image specified by "
+                                 "'--source' and '--registry' is a valid tag:\n\t%s\n", "{}/{}"
+                                 .format(registry.login_server, source_image))
+        except Exception as unexpected_ex:  # pylint: disable=broad-except
```

Comment:
Is it possible to use a more specific type of exception to avoid broad-except warning?

Reply:
The CLIError seems like the important thing to report to the user.

Reply:
This broad exception is just in case there are any issues in the try block beforehand... You are right and I will try to narrow this to catch any possible issues...